keel-bakery-plugin/src/main/kotlin/com/netflix/spinnaker/config/BakeryConfiguration.kt

```diff
@@ -25,24 +26,24 @@ class BakeryConfiguration {
   @Bean
   fun imageHandler(
     artifactRepository: ArtifactRepository,
+    deliveryConfigRepository: DeliveryConfigRepository,
```

Comment:
If you're going to use both of these repositories here I'd suggest you just wire in the `KeelRepository`